External/Plugins/FlashDebugger/Controls/DataTree/ValueNode.cs

```diff
@@ -274,6 +249,51 @@ public ValueNode(string text, Value value)
             m_Value = value;
         }
 
+        private string Escape(string text)
+        {
+            text = text.Replace("\\", "\\\\");
+            text = text.Replace("\"", "\\\"");
+            text = text.Replace("\0", "\\0");
+            text = text.Replace("\a", "\\a");
+            text = text.Replace("\b", "\\b");
+            text = text.Replace("\f", "\\f");
+            text = text.Replace("\n", "\\n");
+            text = text.Replace("\r", "\\r");
+            text = text.Replace("\t", "\\t");
+            text = text.Replace("\v", "\\v");
+            if (text.Length > 65533)
+                text = text.Substring(0, 65533 - 5) + "[...]";
+            return text;
+        }
+
+        private string CleanTypeClassPaths(string qualifiedName)
```

Comment:
Well, that code has been there "forever". I only moved it to another place since I didn't like it to be in the middle of properties.